crates/anvil/core/src/eth/mod.rs

```diff
@@ -768,6 +768,10 @@ pub enum EthRequest {
         serde(rename = "anvil_removePoolTransactions", with = "sequence")
     )]
     RemovePoolTransactions(Address),
+
+    // Reorg the chain
+    #[cfg_attr(feature = "serde", serde(rename = "anvil_reorg",))]
+    Reorg(u64, u64, #[cfg_attr(feature = "serde", serde(default))] Vec<(TransactionRequest, u64)>),
```

Comment:
unclear what all the values are, should we convert this to a single object instead? I'd prefer an object.

I'd also like to see a sanity test here

Reply:
Good callout. Will convert to object to provide more type safety.